src/HeatmapView.js

```diff
@@ -49,7 +49,7 @@ class HeatmapView extends React.Component {
         isLoading: false,
         hasError: {
           description: `There was a problem communicating with the server. Please try again later.`,
-          name: e.name,
+          name: e.geneName,
```

Comment:
This is the `name` field from the error object. I think you mistakenly renamed it.